addons/l10n_in_edi/views/edi_pdf_report.xml

```diff
@@ -1,76 +1,29 @@
-                    <img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' %
-                    ('QR', l10n_in_einvoice_json['SignedQRCode'], 150, 150)"/>
-                </div>
-            </div>
-        </div>
-        <div class="col-6 text-right mb4" position="attributes">
-            <attribute name="class">col-3 text-right mb4</attribute>
-        </div>
-    </template>
-
-    <template id="external_layout_bold_inherit_l10n_in_edi" inherit_id="web.external_layout_bold">
-        <img t-if="company.logo" position="after">
-            <strong style="font-size: 10px;" t-if="l10n_in_einvoice_json">IRN:<span t-esc="l10n_in_einvoice_json['Irn']"/></strong>
-        </img>
-        <xpath expr="//div[hasclass('row')]/div[hasclass('col-6')]" position="after">
-            <div class="col-2 text-right" t-if="l10n_in_einvoice_json">
-                <img class="mt-2" t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' %
-                ('QR', l10n_in_einvoice_json['SignedQRCode'], 150, 150)"/>
-            </div>
-        </xpath>
-        <xpath expr="//div[hasclass('row')]/div[hasclass('col-6')]" position="attributes">
-            <attribute name="class">col-5</attribute>
-        </xpath>
-        <xpath expr="//div[@name='company_address']" position="attributes">
-            <attribute name="class">col-4</attribute>
-        </xpath>
-    </template>
-
-    <template id="external_layout_striped_inherit_l10n_in_edi" inherit_id="web.external_layout_striped">
-        <xpath expr="//div[hasclass('company_address')]" position="after">
-            <div class="text-right" t-if="l10n_in_einvoice_json">
-                <img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' %
-                ('QR', l10n_in_einvoice_json['SignedQRCode'], 150, 150)"/><br/>
-                <span style="font-size: 10px;">IRN:<span t-esc="l10n_in_einvoice_json['Irn']"/></span>
-            </div>
-        </xpath>
-    </template>
-
     <template id="l10n_in_einvoice_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
         <xpath expr="//div[@id='informations']" position="inside">
             <t t-set="l10n_in_einvoice_json" t-value="o._get_l10n_in_edi_response_json()"/>
             <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="ack_no">
-                <strong>Ack. No:</strong>
+                <strong>Acknowledgement:</strong>
                 <p class="m-0" t-out="l10n_in_einvoice_json['AckNo']"/>
+                <p t-out="l10n_in_einvoice_json['AckDt']"/>
             </div>
-            <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="ack_date">
-                <strong>Ack. Date:</strong>
-                <p class="m-0" t-out="l10n_in_einvoice_json['AckDt']"/>
+        </xpath>
+        <xpath expr="//div[@id='total']/div[1]" position="attributes">
```

Comment:
@jva-odoo You seem to be changing this for all invoices, not just for the Indian ones.

Reply:
okay, let me try to do it only for Indian ones.
but that handled in `t-attf-style` and `t-attf-class` not here

Reply:
@jco-odoo now check